index.html

```diff
@@ -61,19 +61,19 @@ <h1>
       </section>
       <section class="readings-container">
         <article class="readings">
-          <h2 class="readings-title">Temperature</h2>
+          <h2 class="readings-title" id="temperature">Temperature</h2>
           <p class="readings-value">&deg;C</p>
         </article>
         <article class="readings">
-          <h2 class="readings-title">Pressure</h2>
+          <h2 class="readings-title" id="pressure">Pressure</h2>
           <p class="readings-value">1013 hPa</p>
         </article>
         <article class="readings">
-          <h2 class="readings-title">Humidity</h2>
+          <h2 class="readings-title" id="humidity">Humidity</h2>
```

Comment:
An id should be given to the p tag because it contains the readings value not the h2 title tag, so you can edit it later through javascript.